### ISSUE TITLE
Should we enable Style/MultilineWhenThen

### ISSUE THREAD

**Issue by thijsnado** [testdouble/standard#189]
Standard does not enable this rule. I figure for sake of consistency it makes sense? https://docs.rubocop.org/rubocop/cops_style.html#stylemultilinewhenthen for documentation, but the TLDR is don't use the `then` part of when unless specifying one liners.

**Comment by searls** [comment]
Yeah I don't see why we wouldn't want this one, Thijs. Want to send a quick PR?
